docs/scos/dev/keeping-a-project-upgradable/upgradability-guidelines/php-version.md

```diff
@@ -0,0 +1,77 @@
+---
+title: PHP version
+description: Reference information for evaluator tools.
+template: howto-guide-template
+---
+
+The allowed and consistent PHP version is used in different project parts.
+
+## Problem description
+
+The PHP version is declared in different configuration files and used by dependencies.
+Need to be sure that all those items have the consistent and supported by spryker PHP version.
+
+PHP versions are checked in:
+- `composer.json`
+- Config deploy files `deploy.**.yml`
+- Installed PHP version
+- SDK php versions
```

Comment:
It can be confusing for the customer. It doesn't explain here what is SDK and why it is there. Probably, we can add a link and write `Spryker SDK` to be more precise, because Spryker also offers Docker SDK and it can be confusing.